graphql-hive.gemspec

```
@@ -24,9 +24,9 @@ Gem::Specification.new do |spec|
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{\A(?:test|spec|features)/}) }
   end
 
-  spec.add_dependency 'graphql', '~> 2.0.9'
+  spec.add_dependency 'graphql', '~> 2.3'
```

Can we just pin this to `~> 2` instead so we don't run into it in the future?

```suggestion
  spec.add_dependency 'graphql', '~> 2'
```